cypress/component/QueryBuilder/QueryBuilder.spec.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from 'react'
+import { QueryBuilder } from '@toptal/picasso-query-builder'
+import type {
+  RuleGroupTypeAny,
+  RuleType,
+  ValidationResult,
+} from '@toptal/picasso-query-builder'
+import { Container } from '@toptal/picasso'
+import { useNotifications } from '@toptal/picasso/utils'
+
+const submitSuccessMessage = 'Successfully submitted'
+const firstnameValidationMessage = 'Firstname is required'
```

Comment:
NIT: https://english.stackexchange.com/questions/78894/firstname-or-first-name

```suggestion
const firstNameValidationMessage = 'First name is required'
```